service/src/test/java/uk/gov/hmcts/reform/fpl/service/FeatureToggleServiceTest.java

```diff
@@ -22,143 +34,246 @@
 import static uk.gov.hmcts.reform.fpl.enums.AllocatedJudgeNotificationType.NOTICE_OF_PROCEEDINGS;
 import static uk.gov.hmcts.reform.fpl.enums.AllocatedJudgeNotificationType.SDO;
 
-@ExtendWith(SpringExtension.class)
-public class FeatureToggleServiceTest {
+@ExtendWith(MockitoExtension.class)
+class FeatureToggleServiceTest {
 
     private static final String LD_USER_KEY = "test_key";
 
-    @MockBean
-    private LDClient ldClient;
+    private static LDClient ldClient = Mockito.mock(LDClient.class);
+    private static FeatureToggleService service = new FeatureToggleService(ldClient, LD_USER_KEY);
 
-    private FeatureToggleService featureToggleService;
+    @Captor
+    private ArgumentCaptor<LDUser> ldUser;
 
-    @BeforeEach
-    void setup() {
-        featureToggleService = new FeatureToggleService(ldClient, LD_USER_KEY);
+    @AfterEach
+    void tearDown() {
+        reset(ldClient);
     }
 
     @ParameterizedTest
     @ValueSource(booleans = {true, false})
-    public void shouldMakeCorrectCallForXeroxPrinting(Boolean toggleState) {
+    void shouldMakeCorrectCallForXeroxPrinting(Boolean toggleState) {
         givenToggle(toggleState);
 
-        assertThat(featureToggleService.isXeroxPrintingEnabled()).isEqualTo(toggleState);
+        assertThat(service.isXeroxPrintingEnabled()).isEqualTo(toggleState);
+        verify(ldClient).boolVariation(eq("xerox-printing"), ldUser.capture(), eq(false));
+
+        LDUser expectedLdUser = getLdBuilder().build();
 
-        verify(ldClient).boolVariation(eq("xerox-printing"), any(LDUser.class), eq(false));
+        assertThat(ldUser.getValue().getCustomAttributes()).isEqualTo(expectedLdUser.getCustomAttributes());
```

Comment:
Do we want to keep this capture now that we are doing the essentially the same thing in the `shouldNotCarryAttributesBetweenRequests` test (same for other tests as well)?